deel-2-flexbox-layout/opdracht-5/styles/styles.css

```diff
@@ -1 +1,48 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+/*Schrijf hier jouw CSS*/
```

Comment:
Algemeen: Elementen vullen niet de volledige hoogte van de browser. Ik weet alleen niet waarom dat is.